vault/wrapping.go

```diff
@@ -309,16 +309,57 @@ DONELISTHANDLING:
 	return nil, nil
 }
 
-// ValidateWrappingToken checks whether a token is a wrapping token.
-func (c *Core) ValidateWrappingToken(ctx context.Context, req *logical.Request) (bool, error) {
+// validateWrappingToken checks whether a token is a wrapping token. The passed
+// in logical request will be updated if the wrapping token was provided within
+// a JWT token.
+func (c *Core) ValidateWrappingToken(ctx context.Context, req *logical.Request) (valid bool, err error) {
+	defer func() {
+		// Perform audit logging before returning if there's an issue with checking
+		// the wrapping token
+		if err != nil || !valid {
+			// Get non-HMAC'ed request data keys
+			var nonHMACReqDataKeys []string
```

Comment:
This isn't necessary is it?  The mount entry will always be sys, which isn't user-mountable or tuneable, thus it will never have non-hmac keys.